socrates/lib/site/views/schedule.jade

```diff
@@ -93,14 +98,16 @@ block content
         | #{''} from previous years.
       p Facilitation will be in English. However, it will be possible to run sessions in other languages.
 
-      h2 Evening activities
-      p
-        | Participants organize evening activities. In the previous years there
-        | were a multitude of things going on in the evenings, from music (some participants
-        | brought instruments, songbooks etc., one year they even wrote a song about the conference) over board games to #{''}
-        +extlink('https://github.com/janernsting/maexchen', 'live-hacking programs to play mia against each other')
-        |. And of course there's always plenty of people to just talk to, or continue discussions from the day.
-      p
-        | Check out the #{''}
-        a(href='/wiki/2015/evening-activities') wiki
-        |  page to see what is currently being proposed. Don't forget to contribute your ideas.
+      //
+        TODO include again after wiki update
+        h2 Evening activities
```

Comment:
This is a block comment in jade. At least to my understanding - I'm not familiar with jade too much.